### PR TITLE
Issue forms

### PR DESCRIPTION
**Pull request by 4ydan**
In case we wanna use Issue forms, I wrote a quick template to have a look at its possibilities. It provides a more user friendly way of writing issues. [doc.github](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) 
Issue: #533

## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thank you for creating the form. We can try using it but some improvements are needed.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        Before creating a **bug report**, please check the following:
+          * Ensure you're running the latest PermaplanT release.
+          * Ensure there isn't an existing issue for your bug. If there is, leave a comment on the existing issue.
+
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: Some short sentences.
+    validations:
+      required: true
```
[the remark this answers is not in the document]
Comment (4ydan) [reply .github/ISSUE_TEMPLATE/bug_report.yml:17]: Yes, like to give an idea of what the bug is about. Like: "I cant select a plant or when I open the map I cant do this and that and something else happens"

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      description: If you're not on release, provide the commit hash
+      placeholder: 0.2.1
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Operating System
+      description: Which OS with which version was used?
+      placeholder: Windows 11
```
[the remark this answers is not in the document]
Comment (4ydan) [reply .github/ISSUE_TEMPLATE/bug_report.yml:52]: What about frontend bugs on the webpage. I mean in that case the OS wont matter that much but might be good to know.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (markus2330) [reply .github/ISSUE_TEMPLATE/bug_report.yml:17]: This has quite some overlap to the other points. If we don't require it, I am okay with it, though.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (markus2330) [reply .github/ISSUE_TEMPLATE/bug_report.yml:52]: Ahh, ok, then forget my suggestion.

> I mean in that case the OS wont matter that much but might be good to know.

I agree, fonts and stuff differ after all.

**Comment by markus2330** [comment]
Bug report works nice but left-over issues doesn't seem to work?

**Comment by 4ydan** [comment]
The enhancement template is not showing, I tested it on a prive repo but seems it was only cached in my browser😢

**Pull request merged**